### PR TITLE
deprecation changes

### PR DESCRIPTION
**Pull request by cjwilburn**
@alairock Not sure if even though you're no longer able to maintain this, that you're monitoring deprecations. At any rate, merging this, and publishing the changes should take care of the Atom warnings.

### CODE REVIEW

**Comment by cjwilburn** [comment]
Adresses #22.

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "atom": ">0.150.0"
   },
   "dependencies": {
+    "atom-space-pen-views": "^2.0.3"
```
Comment (alairock) [inline package.json]: Shouldn't this have a comma after it?

**Comment by alairock** [comment]
I'd be happy to merge this after the line commented on above has been addressed.

**Comment by cjwilburn** [comment]
Yup. Apologies for that. I've been kind of multi-tasking and didn't fully test the package file. This has been fixed.

**Comment by cjwilburn** [comment]
Wondering if you've published the fixes yet. I'm still getting deprecation warnings in Atom.

**Comment by alairock** [comment]
Committed to master but I just deployed a new version. Let me know if that works for ya.

**Comment by cjwilburn** [comment]
Hmm. The update didn't seem to take. And you released it through `apm publish` ?

**Comment by alairock** [comment]
Not sure what happened there. Should be out now. Let me know if you have any further issues with it.

**Comment by cjwilburn** [comment]
It's up now. Thanks a bunch.

**Pull request merged**